modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/UserDisplayContext.java

```diff
@@ -158,7 +158,11 @@ public List<UserGroupRole> getOrganizationRoles() throws PortalException {
 	public List<Organization> getOrganizations() throws PortalException {
 		if (_selUser != null) {
 			if (!_initDisplayContext.isFilterManageableOrganizations()) {
-				return _selUser.getOrganizations();
+				List<Organization> organizations = _selUser.getOrganizations();
+
+				organizations.addAll(_getParentOrganizations(organizations));
+
+				return organizations;
 			}
 
 			return UsersAdminUtil.filterOrganizations(
```

Comment:
Can you help me when this part might be used? As I see your change only affects the case when `isFilterManageableOrganizations` is false. Is it not possible that the extended list of groups will be needed in the other case?

Reply:
Hi @oliverkecskemety !

It looks like this variable filterManageableOrganizations will be true[ if the user is not the company administrator or does not have permission to see the organizations listing](https://github.com/liferay-appsec/liferay-portal/blob/b125938e0885a034f78ffc238b8483e0f8d1975f/modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/InitDisplayContext.java#L60). With these restrictions in mind, is it still needed?

Reply:
Hi @rijema, sorry I forgot to comment here. @stian-sigvartsen raised some concerns regarding the inheritance of roles between organizations. We are not sure if it should work the way it works right now and I try to reach out to the former U&SM Team to clarify things. Thanks for you patience!

Reply:
Hi @oliverkecskemety ! any updates on this ?

Reply:
Hi @MirnaGama,

Sorry for the delay and thank you for your patience! I managed to validate that this is in fact the correct behaviour. However, I learned that [there is a property](https://github.com/liferay/liferay-portal/blob/7f2c19002f7bb46b6f3b86c707487414e1619074/portal-impl/src/portal.properties#L2473-L2483) which you can set to disable inheritance between organizations. Can you check please if your change handles that correctly?

Plus I think it would be better if both cases (the currently changed and the `UsersAdminUtil.filterOrganizations`) would use the whole set of organizations. Please change that.

Thank you!

Reply:
Hi @oliverkecskemety 

Thank you for your feedback. I'm currently working to adapt the source code to include this property scenario case.